lib/mayaUsd/utils/utilSerialization.cpp

```diff
@@ -97,6 +97,10 @@ bool saveRootLayer(SdfLayerRefPtr layer, const std::string& proxy)
     fp = TfStringReplace(fp, "\\", "/");
 #endif
 
+    if (savePathAsRelative) {
+        fp = UsdMayaUtilFileSystem::getPathRelativeToMayaSceneFile(fp);
+    }
```

Comment:
As mentioned before, getPathRelativeToMayaSceneFile currently can return an empty path, so we should protect against it here or within that function.